src/app/container/descriptors/descriptors.component.spec.ts

```diff
@@ -22,8 +22,7 @@ import { ContainersStubService, ContainerStubService, EntryStubService, GA4GHV20
 import { ContainerService } from '../../shared/container.service';
 import { DescriptorService } from '../../shared/descriptor.service';
 import { FileService } from '../../shared/file.service';
-import { EntriesService, GA4GHV20Service } from '../../shared/openapi';
```

Comment:
Do we know why these re-arrangements are occurring? 
Are imports not managed by prettier?
@kathy-t @coverbeck

Reply:
Ying mentioned in the PR description that she intentionally did this, but that's true, we should make sure prettier won't undo them the next time somebody edits a file.